builtin/providers/google/resource_container_cluster.go

```diff
@@ -25,8 +25,9 @@ func resourceContainerCluster() *schema.Resource {
 		Schema: map[string]*schema.Schema{
 			"master_auth": &schema.Schema{
 				Type:     schema.TypeList,
-				Required: true,
+				Optional: true,
 				ForceNew: true,
+				MaxItems: 1,
```

Comment:
Whoops, missed this earlier- you'll also need to set this to computed to get tests without master_auth to pass, since the server does return master_auth info.